engine/gamesys/src/gamesys/components/comp_sound.cpp

```diff
@@ -200,62 +207,63 @@ namespace dmGameSystem
         return dmGameObject::UPDATE_RESULT_OK;
     }
 
-    dmGameObject::UpdateResult CompSoundUpdate(const dmGameObject::ComponentsUpdateParams& params, dmGameObject::ComponentsUpdateResult&)
+    static dmGameObject::UpdateResult CompSoundUpdate(const dmGameObject::ComponentsUpdateParams& params, dmGameObject::ComponentsUpdateResult&)
     {
         dmGameObject::UpdateResult update_result = dmGameObject::UPDATE_RESULT_OK;
         SoundWorld* world = (SoundWorld*)params.m_World;
         DM_PROPERTY_ADD_U32(rmtp_Sound, world->m_Components.Size());
         for (uint32_t i = 0; i < world->m_Entries.Size(); ++i)
         {
             PlayEntry& entry = world->m_Entries[i];
-            if (entry.m_SoundInstance != 0)
+            if (!entry.m_SoundInstance)
+                continue;
```

Comment:
This section is only really that I prefer early-outs, as opposed to long indented if statements